test/pspm_get_superclass.m

```diff
@@ -47,7 +47,12 @@ function valid_datafile(this)
             fprintf('\n');
             
             for k = 1:numel(this.testcases)
-                [sts, import, sourceinfo] = this.fhandle(this.testcases{k}.pth, this.testcases{k}.import);
+
+                if (isfield(this.testcases{k}, 'extra_args'))
+                    [sts, import, sourceinfo] = this.fhandle(this.testcases{k}.pth, this.testcases{k}.import,this.testcases{k}.extra_args{1});
+                else;
+                    [sts, import, sourceinfo] = this.fhandle(this.testcases{k}.pth, this.testcases{k}.import);
+                end;

```

Comment:
What does this do ?

Reply:
This is because I changed the function signature of the pspm_get_text so I needed the superclass to be able to handle extra args. But should be able to remove once I make the pspm_get_text the same as the others again